README.md

```diff
@@ -1,24 +1,111 @@
-# README
+# Expense App (Budget App)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
 
-Things you may want to cover:
+> A mobile web application where you can manage your budget
 
-* Ruby version
+![expense](https://user-images.githubusercontent.com/64914462/144418226-3ba15144-4633-471b-a880-459381fe31da.png)
 
-* System dependencies
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Configuration
+## Built With
 
-* Database creation
+- Ruby on Rails
+- PostgreSQL
 
-* Database initialization
+## Live Link
```

Comment:
- [ ] please add a video recording as requested in the project guidelines

Reply:
Video added to the readme but I already had those in my pull request

Reply:
This change has been implemented well